components/serpapi/serpapi.app.mjs

```diff
@@ -1,11 +1,58 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "serpapi",
-  propDefinitions: {},
+  propDefinitions: {
+    engine: {
+      type: "string",
+      label: "Engine",
+      description: "Engine to send the query to",
+      options: constants.ENGINES,
+    },
+    q: {
+      type: "string",
+      label: "Query",
+      description: "The query or queries you want to search. When passing multiple queries you need to use a comma `,` to separate them",
```

Comment:
If there can be multiple queries, shouldn't this be an array (`string[]`) for better usability in the UI?

They can then be `.join`ed when sent to the API